### ISSUE TITLE
rio stage -e creates a new app instead of a version

### ISSUE THREAD

**Issue by lalyos** [rancher/rio#673]
**Describe the bug**
`rio stage --image` creates a new version of an existing app while 
`rio stage -e` creates a new app

**To Reproduce**
1.  create the app: `rio run --name web -p 80  nginx:1.16`
2. create a new version: `rio stage --image=nginx:1.17 web:v2`
3. create a third version by hand editing image: `rio stage -e web:v3`

for the app/version column:
- step 2. creates `web/v2 ` 
- step 3. creates: `web-v3/v3` 

see `rio ps`:
```
NAME      IMAGE        ENDPOINT                                              SCALE     APP/VERSION    WEIGHT    CREATED              DETAIL
web-v3    nginx:1.17   https://web-v3-v3-default.iwontshow.on-rio.io:32364      1         web-v3/v3      0%        About a minute ago   
web-v2    nginx:1.17   https://web-v2-default.iwontshow.on-rio.io:32364         1         web/v2         0%        About a minute ago   
web       nginx:1.16   https://web-264c9f52-default.iwontshow.on-rio.io:32364   1         web/264c9f52   100%      About a minute ago  
```
**Expected behavior**

`rio stage -e web:v3` should create `web/v3` instead of `web-v3/v3`

**Kubernetes version & type (GKE, on-prem)**: `kubectl version`
```
Client Version: v1.16.2
Server Version: v1.14.7-gke.14
```
Type:
**Rio version**: `rio info`
```
Rio Version: v0.6.0-alpha.2 (d9675e43)
Rio CLI Version: v0.6.0-alpha.2 (d9675e43)
System Namespace: rio-system
Wildcard certificates: true
```
**Additional context**
`rio systemlog`
output:
```

```

**Comment by lalyos** [comment]
@vincent99 pointed out on slack, that there is workaround: one can add `app` and `version` to specification:
```diff
spec:
+  app: web
+  version: v3
+  image: nginx:1.17
   ports:
   - port: 80
     targetPort: 80
   weight: 100
```

**Comment by StrongMonkey** [comment]
Available to test in v0.6.0-alpha.3.

**Comment by StrongMonkey** [comment]
Available to test in v0.6.0-alpha3.

**Comment by rancher-max** [comment]
Fixed in v0.6.0-alpha.3.

A few notes:
1. `rio stage -e`is now for environment variables. Use `rio stage --edit` for editing.
2. Setting certain values does not work when using the `--edit` flag. For example, setting `spec.weight` above 0 reverts back to 0, and passing `rio stage -e foo=bar --edit` does not properly set the environment variables.
